### PR TITLE
PA-5723 | Onboard to devops-pipeline and create Service Image

### PR DESCRIPTION
**Pull request by goutamjaiswal**
**Motivation**
Creating Service Image for devops-saml-proxy

**Proposed changes**
Update dockerfile

**Alternatives considered**
N/A

**Testing steps**
Test with sre-blackbox-dev
image

### CODE REVIEW

File: .acquia/pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      file: "Dockerfile"
+      context: "."
+
+# This step performs container image security scan in Jfrog Xray via Artifactory
+# Build will fail whenever a vulnerability is found unless set to true
+security_scan:
+  scanner: orca
+  ignore_failures: false
+
+
```
Comment (AbhishekRatnawat) [inline .acquia/pipeline.yaml:26]: extra lines
```suggestion
```

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=ruby:2.7-alpine
+ARG BASE_IMAGE=ruby:3.3-alpine
 ARG APP_ROOT=/app
 
 FROM $BASE_IMAGE AS build-env
```
Comment (AbhishekRatnawat) [inline Dockerfile:4]: Do we need `AS build-env` if it's not being used.

Comment (goutamjaiswal) [reply Dockerfile:4]: Yes @AbhishekRatnawat It is used at line #25

Comment (AbhishekRatnawat) [reply Dockerfile:4]: Oh fine, I got confused!

**Review by Harshal0908 — APPROVED** [review]
LGTM

File: .acquia/pipeline.yaml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  service_image:
+    - name: devops-saml-proxy
+      file: "Dockerfile"
+      context: "."
+
+# This step performs container image security scan in Jfrog Xray via Artifactory
+# Build will fail whenever a vulnerability is found unless set to true
+security_scan:
+  scanner: orca
+  ignore_failures: false
```
Comment (amanjoshicodes) [inline .acquia/pipeline.yaml:24]: Add new line at EOF

**Review by amanjoshicodes — APPROVED** [review]
Minor suggestion, rest LGTM

**Review by AbhishekRatnawat — APPROVED** [review]
Nice!

File: .acquia/pipeline.yaml (hunk 3 above)
Comment (goutamjaiswal) [reply .acquia/pipeline.yaml]: Addressed

**Pull request merged**